plugin/helpers_plugin.go

```diff
@@ -54,3 +54,16 @@ func (p *HelpersImpl) ensureServerVersion(required string) error {
 	}
 	return nil
 }
+
+// GetPluginAssetURL builds a URL to the resource from asset directory.
+func (p *HelpersImpl) GetPluginAssetURL(pluginID, asset string) (*url.URL, error) {
```

Comment:
Can we add this function to the `Helpers` interface? Without the change to the interface, no plugin can actually use this method .

Reply:
@lieut-data I will work on that, thanks for the feedback.